src/pudl/etl/glue_assets.py

```diff
@@ -0,0 +1,75 @@
+"""FERC and EIA glue assets."""
+from dagster import AssetOut, Output, multi_asset
+
+import pudl
+from pudl.metadata.classes import Package
+
+logger = pudl.logging_helpers.get_logger(__name__)
+
+
+# TODO (bendnorman): Currently loading all glue tables. Could potentially allow users
+# to load subsets of the glue tables, see: https://docs.dagster.io/concepts/assets/multi-assets#subsetting-multi-assets
+# Could split out different types of glue tables into different assets. For example the cross walk table could be a separate asset
+# that way dagster doesn't think all glue tables depend on generators_entity_eia, boilers_entity_eia.
```

Comment:
Good question! No, FK constraints on the database are disabled during table loading because the asset dependencies don't load the tables in an order that satisfies the FK relationships. However, the FK relationships can be checked after all of the data has been loaded into the database.  The `SQLiteIOManager.check_foreign_keys()` method checks the foreign key relationships and returns any errors. This can be run as a test in our CI.

Reply:
oh that's super nice! So it will check everything once loaded and return all of the errors? Instead of just the first one that pops up? That would be a huge process improvement imo